discretize/mixins/vtkModule.py

```diff
@@ -568,10 +600,32 @@ def readVTK(TensorMesh, fileName, directory=''):
         tuple
             (TensorMesh, modelDictionary)
         """
-        fname = os.path.join(directory, fileName)
+        fname = os.path.join(directory, filename)
         # Read the file
         vtrReader = vtkXMLRectilinearGridReader()
         vtrReader.SetFileName(fname)
         vtrReader.Update()
         vtrGrid = vtrReader.GetOutput()
-        return TensorMesh.vtkToTensorMesh(vtrGrid)
+        return TensorMesh.vtk_to_tensor_mesh(vtrGrid)
+
+
+    @classmethod
+    def read_vtk(TensorMesh, filename, directory=''):
```

Comment:
same question here about preferred using interaction

Reply:
`readVTK` has been around for much longer than the rest of this code so switching to pep8 here will likely affect a lot more external code

Reply:
that is totally fair, we can give people time to adapt external code :)

Reply:
I went ahead and added the deprecation warning in b4370a7

Should I take it out?